src/main/java/com/springdata/service/PersonServiceImpl.java

```diff
@@ -0,0 +1,50 @@
+package com.springdata.service;
+
+import com.springdata.model.PersonEntity;
+import com.springdata.repository.PersonRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@AllArgsConstructor(onConstructor = @__(@Autowired))
+public class PersonServiceImpl implements PersonService {
+    private PersonRepository repository;
+
+    @Override
+    public void createPerson(PersonEntity person) {
+        repository.save(person);
+    }
+
+    @Override
+    public void updatePerson(PersonEntity persin) {
+        repository.save(persin);
+    }
+
+    @Override
+    public void deletePerson(Long id) {
+        repository.deleteById(id);
+    }
+
+    @Override
+    public PersonEntity findPersonById(Long id) {
+        if (repository.findById(id).get()!=null) {
```

Comment:
get() did not return null anyway, instead, you will get an exception.
Please fix this code and make it more elegant. 
https://www.baeldung.com/java-optional